algrtm/src/components/Header.js

```diff
@@ -0,0 +1,82 @@
+"use client";
+import Link from "next/link";
+import React from "react";
+import Button from "./shared/Button";
+import Image from "next/image";
+
+export default function Header() {
+  return (
+    <header className="z-[999] relative flex w-full justify-between items-center border-b-[1px] border-[#F5F3F5] border-b-solid p-4 sm:px-[44px] sm:py-4 lg:px-[5.88rem] lg:py-4">
```

Comment:
In Figma, the px for the larger devices should be `5.875rem` and you can also define custom values [docs](https://tailwindcss.com/docs/padding#using-custom-values). For smaller devices the `sm:px-[44px]` should be in rems and in tailwind it's equal to `px-11`. There is no such property like `border-b-solid`, it's just `border-solid`. Instead of `border-b-[1px]`, you can use `border-b` which has by default `1px` width